### ISSUE TITLE
Users – Component Error is Not Announced or Indicated

### ISSUE THREAD

**Issue by katknow** [omeka/Omeka#1145]
"Description: On the User page, it was found that an error is not announced as expected. When users select the ‘invite’ button without including a ‘User email’, the error text is placed at the top of the page and no where else and the error is not announced automatically. The error being placed at the top of the page is okay, so long as the message is announced automatically with live regions and the error component announces the error state (aria-invalid=""true""). This placement causes an issue for screen reader users who navigate back to the edit field and no error is announced with the component that caused the error. 

Note: While errors are only required to be announced as invalid entries. It is recommended to also add a red outline to the field. (with sufficient contrast)

Expected Results: Edit fields that trigger an error have red text and outline to indicate that it is the cause of the error. Error status messages are announced automatically.

Actual Results: The error text is not announced and the error field is not indicated as having an error, programmatically or visually. "

1.3.1 Info and Relationships; Level A; WCAG 2.0
4.1.3 Status Messages; Level AA; WCAG 2.1
